src/event-decoder/fees-collector/fees-collector.event.topics.ts

```diff
@@ -0,0 +1,35 @@
+import { Address } from '@elrondnetwork/erdjs/out';
+
+export class FeesCollectorEventTopics {
+    private readonly eventName: string;
```

Comment:
make fields public readonly. Don't use getters to get the fields. This will be the standard for all events fields

Reply:
done